common/src/main/scala/explore/common/TargetListGroupQueries.scala

```diff
@@ -14,64 +14,52 @@ import crystal.react.reuse._
 import explore.AppCtx
 import explore.components.graphql.LiveQueryRenderMod
 import explore.implicits._
+import explore.model.AsterismGroup
+import explore.model.ObsIdSet
 import explore.model.ObsSummaryWithConstraints
-import explore.model.TargetEnvGroup
-import explore.model.TargetEnvGroupId
-import explore.model.TargetEnvGroupIdSet
-import explore.schemas.implicits._
+import explore.model.TargetWithId
 import explore.utils._
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.VdomNode
 import lucuma.core.model.Observation
 import lucuma.core.model.Target
-import lucuma.core.model.TargetEnvironment
 import lucuma.schemas.ObservationDB
 import lucuma.schemas.ObservationDB.Types._
 import lucuma.ui.reusability._
 import monocle.Focus
 import monocle.Getter
 
 import scala.collection.immutable.SortedMap
+import scala.collection.immutable.SortedSet
 
 import TargetListGroupQueriesGQL._
 
 object TargetListGroupQueries {
```

Comment:
More pontential `TargetList` -> `Asterism` renames.

Reply:
Yeah. As I noted in the description, I'm going to do more renaming. But, it seemed like the diff was going to be hard enough to follow - file renaming doesn't get tracked well in git. 
 
I will make another commit (in this PR) with a bunch of renaming.

Reply:
:+1: I missed that line in the description